### PR TITLE
g++: update German translation

### PR DESCRIPTION
**Pull request by spicyyboi**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.de/common/g++.md [hunk 1]
```diff
@@ -6,16 +6,24 @@
...
+- Binde Bibliotheken, die sich an einem anderen Pfad als die Quelldatei befinden, mit ein:
 
-`g++ {{quelldatei.cpp}} -o {{output_datei}} -I{{header_pfad}} -L{{bibliotheks_pfad}} -l{{bibliotheks_name}}`
+`g++ {{pfad/zu/quelldatei.cpp}} -o {{pfad/zu/binärdatei}} -I{{pfad/zu/headerdatei}} -L{{pfad/zu/bibliothek}} -l{{bibliotheks_name}}`
+
+- Kompiliere und linke mehrere Quelldateien in eine ausführbare Binärdatei:
+
+`g++ -c {{pfad/zu/quelldatei_1.cpp pfad/zu/quelldatei_2.cpp ...}} && g++ -o {{pfad/zu/binärdatei}} {{pfad/zu/quelldatei_1.o pfad/zu/quelldatei_2.o ...}}`
+
+- Gib die Version von g++ aus:
```
Comment (kbdharun) [inline pages.de/common/g++.md:27]:
```suggestion
- Gib die Version von `g++` aus:
```
Just a small suggestion

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.de/common/g++.md (hunk 1 above)
Comment (spicyyboi) [reply pages.de/common/g++.md]: Sounds good, thanks!

File: pages.de/common/g++.md [hunk 2]
```diff
@@ -6,16 +6,24 @@
...
+`g++ {{pfad/zu/quelldatei.cpp}} -Wall -o {{pfad/zu/binärdatei}}`
 
-- Wähle einen Sprachstandard für das Kompilieren:
+- Wähle einen Sprachstandard (C++98/C++11/C++14/C++17) für das Kompilieren:
 
-`g++ {{quelldatei.cpp}} -std={{C++98|C++11|C++14|C++17}} -o {{output_datei}}`
+`g++ {{pfad/zu/quelldatei.cpp}} -std={{C++98|C++11|C++14|C++17}} -o {{pfad/zu/binärdatei}}`
 
-- Binde Bibliotheken, die sich an einem anderen Pfad als die Quelldatei befinden mit ein:
+- Binde Bibliotheken, die sich an einem anderen Pfad als die Quelldatei befinden, mit ein:
```
Comment (pixelcmtd) [inline pages.de/common/g++.md:19]:
```suggestion
- Binde Bibliotheken, die sich an einem anderen Pfad als die Quelldatei befinden, ein:
```

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**